icu4c/source/i18n/coll.cpp

```diff
@@ -448,6 +448,13 @@ Collator* U_EXPORT2 Collator::createInstance(const Locale& desiredLocale,
 #endif
     {
         coll = makeInstance(desiredLocale, status);
+        // Either returns NULL with U_FAILURE(status), or non-NULL with U_SUCCESS(status)
+    }
+    // The use of *coll in setAttributesFromKeywords can cause causes the NULL check
```

Comment:
Minor nit: "can cause causes" -- the wording feels a bit odd.

Reply:
Ah yes, I changed how I wanted to say this midstream and forgot to delete the old wording. Fixed with another commit.